components/script/dom/html/htmliframeelement.rs

```diff
@@ -631,19 +615,26 @@ impl HTMLIFrameElementMethods<crate::DomTypeHolder> for HTMLIFrameElement {
         Ok(())
     }
 
-    // https://html.spec.whatwg.org/multipage/#dom-iframe-sandbox
+    /// <https://html.spec.whatwg.org/multipage/#dom-iframe-sandbox>
     fn Sandbox(&self, can_gc: CanGc) -> DomRoot<DOMTokenList> {
         self.sandbox.or_init(|| {
             DOMTokenList::new(
```

Comment:
```suggestion
             // The supported tokens for sandbox's DOMTokenList are the allowed values defined
             // in the sandbox attribute and supported by the user agent.
            DOMTokenList::new(
```

Reply:
I've gone ahead and added this to the rustdoc.